packages/cactus-core-api/src/main/typescript/plugin/ledger-connector/i-socket-api-client.ts

```diff
@@ -0,0 +1,25 @@
+import { Observable } from "rxjs";
```

Comment:
@outSH

```suggestion
import type { Observable } from "rxjs";
```

This way the entire rxjs library won't end up in the bundle of the core-api package because the compiler will throw out the import at compilation time (we want to keep the core-api package as light as possible and decoupled from the implementation bundles).

This will also allow you to move `rxjs` into the `devDependency` section of the package.json, I'll make another comment about that.

Reply:
@petermetz Cool, I didn't like including the full package but wasn't aware you can import just the typedefs, thanks! Already changed